package/Dockerfile.shipyard-linting

```diff
@@ -0,0 +1,37 @@
+FROM alpine
+
+ENV DAPPER_HOST_ARCH=amd64 SHELL=/bin/bash \
+    SHIPYARD_DIR=/opt/shipyard
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} \
+    SCRIPTS_DIR=${SHIPYARD_DIR}/scripts
+
+# Requirements:
+# Component        | Usage
+# -------------------------------------------------------------------
+# bash             | Just your basic shell
+# findutils        | Finding executables to compress
+# gitlint          | Git commit message linting
+# grep             | For listing targets
+# make             | Running makefiles inside the container
+# markdownlint     | Markdown linting
+# nodejs           | Used by markdownlint
+# npm              | Installing markdownlint (Removed afterwards)
+# py3-pip          | Installing gitlint (Removed afterwards)
+# shellcheck       | Shell script linting
+# upx              | Compressing executables to get a smaller image
+# yamllint         | YAML linting
+
+RUN apk add --no-cache bash findutils git grep make nodejs shellcheck upx yamllint && \
+    apk add --no-cache --virtual installers npm py3-pip && \
+    npm install -g markdownlint-cli && \
+    pip install gitlint && \
```

Comment:
I think we're supposed to remove npm and pip after these installs?

https://github.com/submariner-io/shipyard/blob/devel/package/Dockerfile.shipyard-dapper-base#L45

Reply:
Or I guess this is handled by `apk del installers`?